resolwe_bio/processes/expression/expression_aggregator.yml

```diff
@@ -45,10 +45,10 @@
     program: |
       {% for e in exps %}
         {% if e.source != exps.0.source %}
-          re-error "All expressions must have the same source."
+          re-error "Genes in all expression data must be annotated by the same genome database (NCBI, UCSC, ENSEMBLE,..)."
```

Comment:
Here, we should use the same logic to list only relevant sources.

Reply:
@hadalin Since I dont't think we are using this process in any of visualizations. So I have skipped listing only relevant sources, because only experienced bioinformatician will see this message. Please correct me if I am wrong?

Reply:
I'd put it there anyway, but it's not my call. @jkokosar?

Reply:
@jkokosar then it probably make sense to go systematically through all processes and change this, I don't think it is many processes left with that kind of Errors?